data/tlg0011/tlg008/__cts__.xml

```diff
@@ -1,15 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" urn="urn:cts:greekLit:tlg0011.tlg008" xml:lang="grc" groupUrn="urn:cts:greekLit:tlg0011">
-   <ti:title xml:lang="lat">Ichneutae</ti:title>
-   <ti:title xml:lang="eng">Tracking Satyrs</ti:title>
-   
+   <ti:title xml:lang="lat">Ichneutae</ti:title>   
    <ti:edition xml:lang="grc" workUrn="urn:cts:greekLit:tlg0011.tlg008" urn="urn:cts:greekLit:tlg0011.tlg008.perseus-grc2">
-      <ti:label xml:lang="eng">Ichneutae, The Oxyrhynchus papyri Part 9 1174</ti:label>
-      <ti:description xml:lang="eng">Sophocles, creator; Grenfell, Bernard P. (Bernard Pyne), 1869-1926, creator; Hunt, Arthur S. (Arthur Surridge), 1871-1934, creator; Grenfell, Bernard P. (Bernard Pyne), 1869-1926, editor; Hunt, Arthur S. (Arthur Surridge), 1871-1934, editor</ti:description>
+      <ti:label xml:lang="lat">Ichneutae</ti:label>
+      <ti:description xml:lang="eng">Sophocles. Oxyrhynchus Papyri, Part 9. Grenfell, Bernard P. (Bernard Pyne); Hunt, Arthur S. (Arthur Surridge), editors.
```

Comment:
Do we do this name expansion elsewhere? I don't think so. It's fine if you'd like but seems non-standard.